KsApi/models/Project.swift

```diff
@@ -119,12 +119,11 @@ public struct Project {
     public let small: String
   }
 
-  public var endsIn48Hours: Bool {
-    return self.dates.deadline - Date().timeIntervalSince1970 <= 60.0 * 60.0 * 48.0
+  public func endsIn48Hours(today: Date = Date()) -> Bool {
```

Comment:
This is now a function that receives `today` as a parameter to make it testable.